### ISSUE TITLE
How to resume training with a different learning rate or else TrainingArguments？

### ISSUE THREAD

**Issue by Bardbo** [huggingface/transformers#21399]
I tried the following settings, which will be covered by the learning rate of checkpoint

```
training_args = TrainingArguments(
    learning_rate=5e-4, # larger than before 5e-5
）
...
trainer.train(resume_from_checkpoint=True)```

**Comment by sgugger** [comment]
Please use the [forums](https://discuss.huggingface.co/) for surch questions as we keep issues for bugs and feature requests only.The resume training functionality is intended in case of instance crash and you should use the same hyperparameters.
